src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -899,41 +899,39 @@ Sentry owner or manager permissions are required to install this integration. Sl
 
 1. In Sentry, navigate to Organization Settings > **Integrations**.
 
-1. At the top of the page, you’ll see the available Global Integrations list, which includes Slack. Click **Install** to integrate Sentry with your Slack workspace(s).
+1. Once on the page, you’ll see the available Global Integrations list, which will include Slack. You can use the search bar to filter for Slack, if it doesn't appear at the top. 
+Click on **Slack** to go to the details page for the integration.
```

Comment:
Suggest: "Click **Slack** to navigate to the details page for this integration."